<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherForm</class>
 <widget class="QMainWindow" name="TeacherForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QComboBox" name="cbbCourse"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblAverage">
        <property name="text">
         <string>Average: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cbbTerm"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblAccountID">
        <property name="text">
         <string>Account ID: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Course Name: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblCount">
        <property name="text">
         <string>Total Count: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Term: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblName">
        <property name="text">
         <string>Name: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="txtName">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="txtAccountID">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="txtCount">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="txtAverage">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="15">
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuset">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="acImport"/>
    <addaction name="acExport"/>
    <addaction name="actionDelete"/>
    <addaction name="acSave"/>
    <addaction name="actionSave_All"/>
    <addaction name="acStatistics"/>
    <addaction name="separator"/>
    <addaction name="acLogout"/>
   </widget>
   <addaction name="menuset"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="acImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="acSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="acStatistics">
   <property name="text">
    <string>Statistics</string>
   </property>
  </action>
  <action name="acLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="acExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
